specifyweb/stored_queries/batch_edit.py

```diff
@@ -1012,6 +1013,7 @@ class BatchEditProps(TypedDict):
     recordsetid: Optional[int]
     session_maker: Any
     fields: List[QueryField]
+    omit_relationships: Optional[bool] = False
```

Comment:
because defaults don't work in typed dict, so this is actually illegal

Reply:
Also, can you try running unit tests locally? it'll be faster, and you actually run specific tests (like just the ones which are erroring / failing). Refer to docs. 
https://docs.djangoproject.com/en/5.1/topics/testing/overview/
It'll be more faster to develop, rather than waiting for 10+ minutes

Reply:
Yeah I've been running them locally but I've been getting some weird errors locally which don't actually happen on the build checks. Didn't know I could test them individually, thanks!